typings/index.d.ts

```diff
@@ -395,15 +394,15 @@ declare module 'discord.js' {
       MESSAGE_REACTION_ADD: 'messageReactionAdd';
       MESSAGE_REACTION_REMOVE: 'messageReactionRemove';
       MESSAGE_REACTION_REMOVE_ALL: 'messageReactionRemoveAll';
+      MESSAGE_REACTION_REMOVE_EMOJI: 'messageReactionRemoveEmoji';
       USER_UPDATE: 'userUpdate';
       PRESENCE_UPDATE: 'presenceUpdate';
       VOICE_STATE_UPDATE: 'voiceStateUpdate';
+      VOICE_SERVER_UPDATE: 'voiceServerUpdate';
```

Comment:
This event was added to constants in #3124 even though it's not used in the lib... I think it should probably be removed, up for discussion

Reply:
The raw data of this event is already accessible through [`Client#ws`](https://discord.js.org/#/docs/main/stable/class/WebSocketManager) and this is not something you need a lot (e.g. if you use an external audio player).

With that in mind, I think it's fine to just remove this line instead of just forwarding it in the lib.

Reply:
You're saying it should be removed from `Constants.js`, correct? Typings should be in sync with the actual file, so I would undo adding this line as well